recipes-bsp/coreboot-utils/amdtool_git.bb

```
@@ -0,0 +1,30 @@
+require coreboot-utils.inc
+
+SUMMARY = "\
+    A utility similar to inteltool, which dumps useful information for \
+    porting a board to coreboot.\
+"
+
+DEPENDS += "pciutils zlib"
+
+EXTRA_OEMAKE = ' \
+               DESTDIR="${D}" \
+               PREFIX="${prefix}" \
+               '
+
+SRC_URI += " \
+    git://github.com:coreboot/coreboot.git;branch=main;protocol=https \
+    file://coreboot-amdtool.patch;name=amdtool-patch \
+"
+
+SRC_URI[amdtool-patch.sha256sum] = "aad05798491213a35e5dffcd4c8b9d173ea4b50722b8ea9847e0cdd179315d6b"
```

```suggestion
    file://coreboot-amdtool.patch \
"
```

You don't need sha256 when using local files

---

fixed: https://github.com/zarhus/meta-coreboot/compare/5ce80399d900fb04dbd48ae97f3828765397d796..bac417c014c37b5b540216bdf7e265d726371960